src/Pass/ImgTagTransformPass.php

```diff
@@ -133,6 +144,19 @@ protected function isSvg(\DOMElement $el)
     }
 
     /**
+     * Detects if the img is a 1x1 pixel. In that case we convert to <amp-pixel> instead of <amp-img>
+     * @param DOMQuery $el
+     * @return bool
+     */
+    protected function isPixel(DOMQuery $el)
+    {
+        if (!$this->setResponsiveImgHeightAndWidth($el)) {
```

Comment:
Are you suggesting to remove the height and width attributes if they were not already set so that `isPixel` doesn't modify attributes of `$el`? Sorry, I don't completely understand.